README.md

```diff
@@ -21,21 +22,34 @@ Or install it yourself as:
 
 ## Usage
 
-The repository is expected to contain one folder per Engine Yard
-environment, each containing files with `.env` extension. For example:
+The config app which is used for storing sensitive data is expected to
+be hosted within the same parent directory on the same git server as
+the software which uses the sensitive data. The sensitive data repo is
+expected to carry the name $APP_NAME_config.git if the software repo
+is named $APP_NAME.git. For example, if sensitive data can be found at
+`git@examplegit.com:companyrepo/software_config.git`, ey_secrets
+expects to find the app repo at
+`git@examplegit.com:companyrepo/software.git`. You can instead provide
+the app's name on Engine Yard using the `-a` option.
```

Comment:
Let's not present the naming scheme as a requirement. Instead let's just have the example be explicit and use `-a`. Then we can mention the naming scheme in a separate remark afterwards, as a way to skip the `-a` option.